build-ps/percona-server-8.0_builder.sh

```diff
@@ -337,7 +337,9 @@ get_sources(){
     sed -i "s:@@REVISION@@:${REVISION}:g" build-ps/percona-server.spec
     sed -i "s:@@RPM_RELEASE@@:${RPM_RELEASE}:g" build-ps/percona-server.spec
     sed -i "s:@@BOOST_PACKAGE_NAME@@:${BOOST_PACKAGE_NAME}:g" build-ps/percona-server.spec
-
+    if [ "x${RHEL}" = "x6" ]; then
+        sed -i "s:-DWITH_ENCRYPTION_UDF=ON:-DWITH_ENCRYPTION_UDF=OFF:g" build-ps/percona-server.spec
+    fi
```

Comment:
we do not support centos6 any more

Reply:
It's one of requirements in the ticket. Maybe some custom builds would be requested on centos 6.
<img width="724" alt="Screenshot 2022-05-20 at 15 43 18" src="https://user-images.githubusercontent.com/25184101/169530649-9161d4fb-28f4-43f2-9879-e8691060e152.png">